go/adbc/driver/snowflake/snowflake_database.go

```diff
@@ -209,7 +209,7 @@ func (d *databaseImpl) SetOptionInternal(k string, v string, cnOptions *map[stri
 	case OptionRegion:
 		d.cfg.Region = v
 	case OptionAccount:
-		d.cfg.Account = v
+		d.cfg.Account = strings.ReplaceAll(v, "_", "-")
```

Comment:
This is the actual change. I had to do linting to get past the precommit checks that I guess were turned on over the weekend?